<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formRegistrarPlatos</class>
 <widget class="QDialog" name="formRegistrarPlatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="PB_Registrar">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Cancelar">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Codigo:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Categoria:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Stock:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Precio:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Descripcion:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="getDescripcion"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="getNombre"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="getCodigo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="getCatego">
      <item>
       <property name="text">
        <string>Entrada</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Segundo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Aperitivo</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="getPrecio">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="getStock">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/recur/imagenes/281776-200.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
